test/BootstrapCommand.js

```diff
@@ -789,4 +794,41 @@ describe("BootstrapCommand", () => {
       bootstrapCommand.runCommand(exitWithCode(0, done));
     });
   });
+
+  describe("environment", () => {
+    let testDir;
+
+    const originalEnv = Object.assign({}, process.env);
+    const mockEnv = {
+      mock_value: 1,
+      NODE_ENV: "lerna-test",
+    };
+
+    beforeEach(() => initFixture("BootstrapCommand/zero-pkgs").then((dir) => {
+      testDir = dir;
+
+      // mock out the ENV to a simpler version for testing
+      process.env = mockEnv;
+    }));
+
+    afterEach(() => {
+      process.env = originalEnv;
+    });
+
+    it("should always honour env vars", (done) => {
+      const bootstrapCommand = new BootstrapCommand([], {});
+      const wantPgkOpts = { cwd: path.join(testDir, "packages"), stdio: STDIO_OPT, env: process.env };
```

Comment:
@evocateur I did that on purpose, since beforEach replaces process.env's value with the mock. It's just the syntax though, so you are right as well, doesn't matter that much.